impl/client/src/main/javascript/web/util/_a11y.js

```diff
@@ -458,6 +458,170 @@ define(["./_focus"], function(focusUtil) {
 
   }
 
+  function makeAccessibleToolbar(elem) {
+    elem.setAttribute("role", "toolbar");
+    elem.setAttribute("tabindex", "0");
+
+    var firstItem = null;
+    var lastItem = null;
+
+    var toolbarItems = [];
+
+    elem.addEventListener("keydown", onKeyPress);
+    elem.addEventListener("focus", onContainerFocus);
+
+    items = elem.querySelectorAll('.toolbarButton');
```

Comment:
Why is there the need to explicitly select toolbar buttons using this class?
Would it be enough to assume that any "tabbable" descendant would be a valid toolbar element?
The only problem I can foresee is if nested tabbable elements exist.

Likewise, and if so, for determining the first, last, next or previous tabbable elements, it would be possible to directly use the `common-ui/util/_focus` API.

If it is really needed to filter tabbable elements to those with a specific selector (including both `toolbarButton` and `hidden` and `disabled`), consider extending the focus API to support that feature (e.g to receive a predicate function that determines if a given tabbable element is a valid candidate).